icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -27,6 +29,74 @@ UBool haveCopyright = TRUE;
 UCPTrieType trieType = UCPTRIE_TYPE_SMALL;
 const char* destdir = "";
 
+// Mask constants for modified values in the Script CodePointTrie, values are logically 12-bits.
+int16_t DATAEXPORT_SCRIPT_X_WITH_COMMON    = 0x0400;
+int16_t DATAEXPORT_SCRIPT_X_WITH_INHERITED = 0x0800;
+int16_t DATAEXPORT_SCRIPT_X_WITH_OTHER     = 0x0c00;
+
+int32_t scxCodePoints[] = {
```

Comment:
Nit: Please add `// TODO(ICU-21821): Replace this with a call to a library function`

Reply:
```suggestion
UChar32 scxCodePoints[] = {
```
... to document that these are code points
(typedef for the same thing though)